api/payload.go

```diff
@@ -168,3 +168,16 @@ type Signal struct {
 type ErrorResponse struct {
 	Message string `json:"msg"`
 }
+
+// LogEntry is the payload for the StreamLog data.
+type LogEntry struct {
+	// Source is the source of the log entry. One of "shim" or "runtime".
+	Source string `json:"source"`
```

Comment:
Or could we just make `Log()` and `Logf()` supplement the `LogEntry` with a call to `os.Getpid()` or similar?